core/controllers/profile.py

```diff
@@ -141,6 +141,26 @@ def get(self):
         user_settings = user_services.get_user_settings(self.user_id)
         user_email_preferences = user_services.get_email_preferences(
             self.user_id)
+
+        creators_subscribed_to = subscription_services.get_all_creators_to_which_learner_has_subscribed( # pylint: disable=line-too-long
```

Comment:
This function name really is very long. Maybe we should change it to get_all_creators_subscribed_to(self.user_id)?